classes/class-wpcom-liveblog-entry.php

```diff
@@ -106,6 +106,22 @@ public function get_timestamp() {
 		return mysql2date( 'G', $this->comment->comment_date_gmt );
 	}
 
+
+	/**
+	 * Retrieve the comment date of the current comment using gmt.
+	 * @param string          $d          Optional. The format of the date. Default user's setting.
+	 * @param int|WP_Comment  $comment_ID WP_Comment or ID of the comment for which to get the date.
+	 *                                    Default current comment.
+	 * @return string The comment's date.
+	 */
+	public function get_comment_date_gmt( $d = '', $comment_ID = 0 ) {
+	        $comment = get_comment( $comment_ID );
+	        if ( '' == $d )
```

Comment:
PHPCS notes:
* wrap the if/else statements in curly braces
* use $commend_id instead of $comment_ID
* Add spaces inside parentheses